src/features/game/scenes/gameManager/GameManager.ts

```diff
@@ -234,6 +252,8 @@ class GameManager extends Phaser.Scene {
       async () => {
         if (this.getPhaseManager().isCurrentPhase(GamePhaseType.EscapeMenu)) {
           await this.getPhaseManager().popPhase();
+        } else if (this.getPhaseManager().isCurrentPhase(GamePhaseType.Dashboard)) {
+          await this.getPhaseManager().swapPhase(GamePhaseType.EscapeMenu);
```

Comment:
I've found that when the dashboard is open, I instinctively press ESC to close it, so the escape menu opening is a little surprising. Not sure if this is just a me problem :(

Reply:
A potential alternative:

Game -> ESC -> Settings -> ESC -> Game (Current)
Game -> TAB -> Dashboard -> TAB -> Game (Current)
Game -> ESC -> Settings -> Tab -> Dashboard (Current)
Game -> TAB -> Dashboard -> ESC -> Game (New)

Reply:
This seems more intuitive to me but I'd like to get your opinions too!

Reply:
We'll go with the alternative suggested above for now, we can get further feedback from the subsequent batch of students if this is an issue :+1: